scopes/scopes.go

```diff
@@ -105,6 +106,12 @@ var (
 	telemetryGatewayPermissions = []Permission{
 		"events",
 	}
+	enterprisePortalPermissions = []Permission{
+		// enterpriseportal.subscriptions.v1.SubscriptionsService
+		"subscription",
+		// enterpriseportal.codyaccess.v1.CodyAccessService
```

Comment:
```suggestion
		// enterprise_portal.subscriptions.v1.SubscriptionsService
		"subscription",
		// enterprise_portal.codyaccess.v1.CodyAccessService
```

do you mean to specify the full scope name or gRPC API name? (if the latter maybe worth calling that out since these two formats look alike?)

Reply:
That's the gRPC name

Reply:
```suggestion
		// gRPC service: enterpriseportal.subscriptions.v1.SubscriptionsService
		"subscription",
		// gRPC service: enterpriseportal.codyaccess.v1.CodyAccessService
```